### ISSUE TITLE
Freezes Foundry.exe when a text box is rolled.

### ISSUE THREAD

**Issue by TheAtrivion** [ultrakorne/better-rolltables#50]
Better Rolltable Freezes my Foundry Client exe. Same thing happens if i roll from a browser but only the Tab that has Foundry freezes. (Maybe an endless loop?)
Foundry starts eating CPU power as soon as the Text box is rolled, It is never displayed in chat.
I can not see why as it doesn't display any error in Console or the log files I've found.
The log files does not record the textbox roll but it does record when im forced to shutdown foundry.
The Rolltable in question is full of my own Items from a Compendium i made and one box for a Text "No fishes bite today".
No roll action can be seen. As soon as i press Roll, everything freezes. 
I've noticed it doesnt happen on newly made Roll tables... The ones that break the game are ones i made a while back, and today decided to add a Text at the bottom for when No fish bite, on my fishing rolltable.
Creating a new rolltable with a Text box doesn't freeze the game when the text is rolled, it just doesnt display the Text.

**Comment by TheAtrivion** [comment]
Ok Addendum.
It works fine aslong as the text isn't longer than "No fish decided to" If i add "No fish decided to be" a lagspike happens and Foundry freezes for a second but then it goes back to normal. 
Im guessing "No fish decided to be eaten that day" is to long a string to add.

**Comment by ultrakorne** [comment]
can you send me the exported table / tables that have issue? here or in discord would be fine.
are you using foundry 0.7?

**Comment by TheAtrivion** [comment]
Hey, Sorry. I've been away from home. 
I did manage to experiment until i found the cause. 
When the Table is registered as a Loot Table, this happens. 
When its registered as a Better Table, it works flawlessly, it prints out text and everything, no freezes. 
I had mine as Loot Tables so it adds the fish rolled straight into a party chest but i decided i wanted a You didnt catch anything text box, so its not a 100% chance of catching fish =P
I used 0.6.5, everything updated this morning. 
I'll send it over Discord, cant add it easy here =)
